concepts/observability-strategy/configurable-monitoring/04-monitoring-api.md

```diff
@@ -53,22 +73,64 @@ spec:
           - hello_world
           - hello/world
 
-  exporter: # only one output, defining no output will fail validation
+  output: # Only one output. Defining no output will fail validation
     prometheusremotewrite:
-      endpoint: "https://my-cortex:7900/api/v1/push"
+      endpoint:
+          value: "https://my-cortex:7900/api/v1/push"
+          valueFrom:
+            secretKeyRef:
+                name: my-config
+                namespace: default
+                key: "endpoint"
       external_labels:
         label_name1: label_value1
         label_name2: label_value2
       tls:
-          insecure: false
-          insecureSkipVerify: true
+        insecure: false
+        insecureSkipVerify: true
+        ca:
+          value: dummy
+          valueFrom:
+              secretKeyRef:
+                name: my-config
+                namespace: default
+                key: "server.crt"
+        cert:
+            value: dummy
+            valueFrom:
+                secretKeyRef:
+                    name: my-config
+                    namespace: default
+                    key: "cert.crt"
+        key:
+            value: dummy
+            valueFrom:
+                secretKeyRef:
+                    name: my-config
+                    namespace: default
+                    key: "client.key"
+    dynatrace:
+      endpoint:
+        value: "https://ab12345.live.dynatrace.com"
+        valueFrom:
+          secretKeyRef:
+              name: my-config
+              namespace: default
+              key: "endpoint"
+      apiToken:
```

Comment:
Maybe we can make it consistent to the "authentication.token" attribute in otlp output